### PR TITLE
🐛 Collections - Fix body styling on smaller screens

### LINKED ISSUE #4614: Collections - Fix list UI on mobile screens

### PR DESCRIPTION
**Pull request by Jord-Gui**
Fixes #4614. The collections table is now scrollable when the width of the screen is too small.

image

**Figure: Collections body on xs screens (320px - 559px)**

### CODE REVIEW

File: packages/tinacms/src/admin/pages/CollectionListPage.tsx [hunk 1]
```diff
@@ -631,267 +631,278 @@ const CollectionListPage = () => {
...
+                                      document.node._sys.breadcrumbs
+                                        .slice(0, -1)
+                                        .join('/')
+
                                     return (
-                                      <tr key={`folder-${document.node.path}`}>
-                                        <td className="pl-5 pr-3 py-3 truncate max-w-0">
+                                      <tr
+                                        key={`document-${document.node._sys.relativePath}`}
+                                        className=""
```
Comment (ncn-ssw) [inline packages/tinacms/src/admin/pages/CollectionListPage.tsx:721]: **question**: Should the className attribute be empty?

Comment (Jord-Gui) [reply packages/tinacms/src/admin/pages/CollectionListPage.tsx:721]: Good catch

Comment (Jord-Gui) [reply packages/tinacms/src/admin/pages/CollectionListPage.tsx]: ✅ Removed

**Review by ncn-ssw — APPROVED** [review]
🦙

**Pull request merged**